app/controllers/idv/capture_doc_status_controller.rb

```diff
@@ -82,9 +82,18 @@ def pending_barcode_attention_confirmation?
     def had_barcode_attention_result?
       if session_result
         flow_session[:had_barcode_attention_error] = session_result.attention_with_barcode?
+        idv_session.had_barcode_attention_error = session_result.attention_with_barcode?
       end
 
       flow_session[:had_barcode_attention_error]
```

Comment:
Since you just set it, can you start returning `idv_session.had_barcode_attention_error` here? Or, since the value is only read here as far as I can tell, do we even need to add it to idv_session? Can we just return `session_result.attention_with_barcode?`

Reply:
Nevermind, I worked through it after I commented... It's polling, so it might set it in one request and read it later.